service/resource/legacyv2/resource.go

```diff
@@ -461,8 +461,9 @@ func (s *Resource) processCluster(cluster v1alpha1.AWSConfig) error {
 	{
 		var err error
 		bucket = &awsresources.Bucket{
-			Name:      bucketName,
-			AWSEntity: awsresources.AWSEntity{Clients: clients},
+			AWSEntity:  awsresources.AWSEntity{Clients: clients},
+			Name:       bucketName,
+			ClusterID:  keyv2.ClusterID(cluster),
```

Comment:
I'd like us to have a function in the `keyv2` package that takes in a custom object and returns a map of keys and values. That will give us one place where we can add standard tags to all AWS resources.